applications/FluidDynamicsApplication/custom_elements/two_fluid_navier_stokes_alpha_method.cpp

```diff
@@ -6,10 +6,7 @@
 //
 //  License:         BSD License
 //                   Kratos default license: kratos/license.txt
-//
-//  Main author:     Daniel Diez
-//  Co-authors:      Ruben Zorrilla
-//
+//  Main author:     Uxue Chasco
```

Comment:
```suggestion

//  Main author:     Uxue Chasco
```
(minor header formatting)